app/controllers/places_controller.rb

```diff
@@ -0,0 +1,74 @@
+#https://jaryl.medium.com/disciplined-rails-models-controllers-the-rails-way-bbc940822136
```

Comment:
is this place controller significant to the map api? or is it looking for stations from the db?

Reply:
db